src/v3/getRates.ts

```diff
@@ -154,24 +152,13 @@ export const getRates: GetRatesFunc = async (params, rightNow) => {
   )
 
   // Update redis with couch data
-  updateProviders([...memoryProviders], {
+  updateProviders([...memoryProviders, ...dbProviders], {
```

Comment:
almost but not quite. you are updating the dbProviders with the data that they returned making a bunch of updates redundant. should instead be:

```
  // Update redis with db and api data
  updateProviders(memoryProviders, {
    targetFiat,
    crypto: new Map([...dbResults.foundCrypto, ...apiResults.foundCrypto]),
    fiat: new Map([...dbResults.foundFiat, ...apiResults.foundFiat])
  }).catch(e => {
    console.error('Error updating providers', e)
  })

  // Update db with api data
  updateProviders(dbProviders, {
    targetFiat,
    crypto: apiResults.foundCrypto,
    fiat: apiResults.foundFiat
  }).catch(e => {
    console.error('Error updating providers', e)
  })
```